src/mcpbr/infrastructure/gcp.py

```
@@ -0,0 +1,921 @@
+        for line in stdout:
+            console.print(line.rstrip())
+
+        # Wait for completion
+        exit_code = stdout.channel.recv_exit_status()
+
+        if exit_code != 0:
+            self._error_occurred = True
+            stderr_output = stderr.read().decode()
+            console.print(f"[red]  Evaluation failed with exit code {exit_code}[/red]")
+            console.print(f"[red]{stderr_output[:2000]}[/red]")
+            raise RuntimeError(f"Evaluation failed: {stderr_output[:500]}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)